src/main/java/mSearch/Config.java

```diff
@@ -19,12 +19,14 @@
  */
 package mSearch;
 
+import java.util.concurrent.atomic.AtomicBoolean;
+
 public class Config {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)